repo_e/sum_od_even.py

```diff
@@ -0,0 +1,17 @@
+# def sum_of_even(integer):
+#     sum=0
+#     for number in integer:
+#         if number%2==0:
+#             sum += number
+#     return sum
+# print(sum_of_even([1,10,4,3]))
+
+def sum_of_even(integer):
+    Even=[]
+    for number in integer:
+        if number%2==0:
+            Even.append(number)
+    return sum(Even)
```

Comment:
```suggestion
    return sum(number for number in numbers if number % 2 == 0)
```
`integer` is not best name for list of numbers, `numbers` would be better